spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -57,6 +57,8 @@
 
 	private boolean ignoreLocalService = true;
 
+	private HostRoutingFilterProperties hostRoutingFilter = new HostRoutingFilterProperties();
```

Comment:
Yeah, traditionally this would be an inner class called `Host` with a field name `host`.